setup.cfg

```diff
@@ -73,6 +73,7 @@ warn_untyped_fields = True
 [flake8]
 max-line-length = 88
 per-file-ignores = __init__.py:F401
-ignore = E203, W503
+ignore = E203, W503, D100, D104, D105, D107, SC200
```

Comment:
I can't seem to find those error codes. Where are they listed?

Reply:
Ds are related to darglint, SCs are related to spellcheck